proof/refine/ARM/Invariants_H.thy

```diff
@@ -2742,7 +2756,13 @@ lemma typ_at_lift_page_table_at'_strong:
   using x
   apply -
   apply (rule P_bool_lift[where P=P])
-  apply (wpsimp wp: hoare_vcg_const_Ball_lift hoare_vcg_bex_lift hoare_vcg_imp_lift
+  apply_trace (wpsimp wp: hoare_vcg_const_Ball_lift hoare_vcg_bex_lift hoare_vcg_imp_lift
+         | fastforce)
+  apply_trace (wpsimp wp: hoare_vcg_const_Ball_lift hoare_vcg_bex_lift hoare_vcg_imp_lift
+         | fastforce)
+  apply_trace (wpsimp wp: hoare_vcg_const_Ball_lift hoare_vcg_bex_lift hoare_vcg_imp_lift
+         | fastforce)
+  apply_trace (wpsimp wp: hoare_vcg_const_Ball_lift hoare_vcg_bex_lift hoare_vcg_imp_lift
```

Comment:
(in `typ_at_lift_page_table_at'_strong`) I would not have expected this proof to change at all. Can you say what breaks in here for MCS?

Reply:
Oh, wait, this is in `ARM/Invariants_H.thy`! Sorry, should have looked closer, I think this is just diagnostics.